sources/assets/shells/aliases.d/cloudmapper

```diff
@@ -0,0 +1 @@
+alias cloudmapper.py='cd /opt/tools/cloudmapper/ && echo "[*] changed dir : cd /opt/tools/cloudmapper/" && ./venv/bin/python3 cloudmapper.py'
```

Comment:
Not possible, those tools rely on messed up paths, hence the `cd`
Instead of looking for resources at `$root/whatever/path/to/file`, they look for `path/to/file` and when running the tool from somewhere else than the tool's directory, things fail
do you have any other idea of how to fix that?

Reply:
Resolving similar comments so that we keep one and treat all others the same

Reply:
Maybe this syntax would do the trick: https://stackoverflow.com/a/10382170

Reply:
It does! Fixing every `cd` aliases now.